### PR TITLE
Adding user awareness text if no data available in my sessions

### LINKED ISSUE #396: Add a Component if no entry found in a route
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
currently, there is no message showing nothing to displayed or no events found for -
`my-sessions/`  ,  `my-sessions/past`  when there are no session proposal for upcoming or past events
and on the home page under upcoming events when there are no upcoming events
image

**Expected behavior:**
there should be a proper message showing if there is any entry found in a route or not
image

### PR DESCRIPTION
**Pull request by harshitagupta30**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adding user awareness text if no data available for my-sessions route.

Deployment link: https://gentle-taiga-30801.herokuapp.com/my-sessions
Fixes #396 

@fossasia/open-event-frontend Please Review. Thanks.

### CODE REVIEW

**Comment by harshitagupta30** [comment]
@niranjan94 @geekyd I have added text for my-tickets as well. Please review it and I guess its good to merge now. 
Deployment link : https://gentle-taiga-30801.herokuapp.com/
Thanks.

File: app/templates/my-tickets/past.hbs [hunk 1]
```diff
@@ -1 +1,5 @@
-Past Data
+{{#if model}}
```
Comment (geekyd) [inline app/templates/my-tickets/past.hbs]: Wrap this inside a row follow the same structure as upcoming sessions.

File: app/templates/my-tickets/saved.hbs [hunk 2]
```diff
@@ -1 +1,5 @@
-Saved Data
+{{#if model}}
```
Comment (geekyd) [inline app/templates/my-tickets/saved.hbs]: Wrap this inside a row follow the same structure as upcoming sessions.

**Comment by harshitagupta30** [comment]
@geekyd I have made the changes as suggested. Please review it again. Thanks

**Comment by harshitagupta30** [comment]
@niranjan94 @CosmicCoder96 @sumedh123 can you please review it?

File: app/templates/my-sessions/past.hbs [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
-  </div>
+  {{#if model}}
+    <div class="sixteen column wide">
+      {{#each model as |session|}}
+        {{session-card session=session}}
+        <div class="ui hidden divider"></div>
+      {{/each}}
+    </div>
+  {{else}}
+    {{t 'No session proposals found for the events'}}
```
Comment (CosmicCoder96) [inline app/templates/my-sessions/past.hbs]: Put this text inside the `sixteen wide column` else it won't be aligned

File: app/templates/my-tickets/index.hbs [hunk 4]
```diff
@@ -1,13 +1,17 @@
...
+          <span>Free Ticket</span>
+          <span>#OD72618129093-104</span>
+          <span>on January 25, 2017 10:30 AM</span>
+        {{/event-card}}
+        <div class="ui hidden divider"></div>
+      {{/each}}
+      {{modals/event-share-modal isOpen=isShareModalOpen event=eventToShare}}
+    </div>
+  {{else}}
+    {{t 'No tickets found for the upcoming events'}}
```
Comment (CosmicCoder96) [inline app/templates/my-tickets/index.hbs]: same here

**Comment by harshitagupta30** [comment]
@CosmicCoder96 I have made the changes suggested by you. I have increased the font size and added ui disabled header class for the text as it was looking very appealing to me. 
@niranjan94 @geekyd @sumedh123  Please have a look at it. 
Deployment link : https://gentle-taiga-30801.herokuapp.com/my-tickets/saved

**Pull request merged**